build_tools/bazel/build_tensorflow.sh

```diff
@@ -70,7 +70,7 @@ fi
 # `bazel test //...` because the latter excludes targets tagged "manual". The
 # "manual" tag allows targets to be excluded from human wildcard builds, but we
 # want them built by CI unless they are excluded with "nokokoro".
-bazel query '//integrations/...' | \
+bazel query //integrations/... + //colab/... | \
```

Comment:
Try also adding //packaging here? Also please update the build description above
```suggestion
bazel query //integrations/... + //colab/... + //packaging/... | \
```